src/utils.js

```diff
@@ -73,6 +76,43 @@ export function parseFiltersToStr(filters) {
   return newFilters.join(',');
 }
 
+export const retrieveDatesFromDateRangeFilterString = filterValue => {
+  let dateRange = {
+    startDate: '',
+    endDate: '',
+  };
+
+  if (filterValue) {
+    const [startDateString, endDateString] = filterValue.split(':');
+    const endDate = moment.utc(endDateString);
+    const startDate = moment.utc(startDateString);
+
+    dateRange = {
+      startDate: startDate.isValid()
+        ? startDate.format(DATE_FORMAT)
+        : '',
+      endDate: endDate.isValid()
+        ? endDate.subtract(1, 'days').format(DATE_FORMAT)
+        : '',
+    };
+  }
+
+  return dateRange;
+};
+
+
+export const makeDateRangeFilterString = (startDate, endDate) => {
+  const endDateCorrected = moment.utc(endDate).add(1, 'days').format(DATE_FORMAT);
+
+  return `${startDate}:${endDateCorrected}`;
+};
+
+export const buildDateRangeQuery = name => values => {
+  const [startDateString, endDateString] = values[0]?.split(':');
```

Comment:
@SergiyVSergiyenko should we make sure that `startDateString` and `endDateString` are present given that `values[0]` is optional?

Reply:
Thanks for the tip, @mkuklis. Fixed.